### PR TITLE
[lipstick] Move screenshottool to its own package

### PR DESCRIPTION
**Pull request by matthewvogt**
screenshottool is the only UI tool provided by this project, so the
name of the package is changed to better describe the content.

### CODE REVIEW

File: rpm/lipstick-qt5.spec [hunk 1]
```diff
@@ -74,16 +74,17 @@ Group:      System/Libraries
...
+Tools for the lipstick package (warning: these tools installed by default).
 
-%package tools-ui
-Summary:    UI-based tools for lipstick
+%package screenshot
+Summary:    Screenshot tool for lipstick
 License:    LGPLv2.1
 Group:      System/Libraries
 Requires:   %{name} = %{version}-%{release}
+Obsoletes:  lipstick-qt5-tools-ui
```
Comment (thp) [inline rpm/lipstick-qt5.spec]: Does this also need a `Provides:` to not break dependencies on the old package?

Comment (matthewvogt) [inline rpm/lipstick-qt5.spec]: Sounds reasonable :)

**Comment by giucam** [comment]
LGTM

**Comment by giucam** [comment]
But, i should add, my understanding of packaging is somewhat limited.

**Comment by jpetrell** [comment]
I liked having access to screenshottool without at the same time installing launcher item (that also currentl lacks app icon). Now we cannot include screenshottool to devices we want to demo that should have clean launcher grid.

**Comment by thp** [comment]
For demo purposes, you can always "rm" the .desktop file. Not as convenient as not having to do anything, but not too problematic.

**Comment by jpetrell** [comment]
True. That's what we used to do when the same issue troubled us couple of years back. Unfortunately many not-so-design-savvy sailors often forget to do these steps before demos.

**Comment by matthewvogt** [comment]
Ok - there's no reason we can't reinstate the separation between UI and backend packages; we just shouldn't have items in the -tools package unless we want them installed by default.

**Comment by matthewvogt** [comment]
@jpetrell - if you want to suggest how things should ideally be packaged, I would recommend commenting on this one: https://bugs.merproject.org/show_bug.cgi?id=976

**Pull request merged**